project/Dependencies.scala

```diff
@@ -1032,15 +1031,15 @@ object Dependencies {
 
   val `kafka-broker-javadsl` = libraryDependencies ++= Seq(
     slf4jApi,
-    "log4j"   % "log4j" % "1.2.17",
+    "org.apache.logging.log4j"   % "log4j-core"                % "2.15.0",
```

Comment:
I'm not sure that it is enough because the cost of migrating from `Log4j 1.x` isn't a zero. 🤔 
https://logging.apache.org/log4j/2.x/manual/migration.html

Reply:
Need to check that we don't use a classes from `Log4j 1.x` directly on code or configs.

Reply:
Yep, that might not be enough. What I did here will only add the log4j 2 as dependency, but the kafka client will still be using the older version. 

We probably need to add the bridge and exclude the log4j 1.x.

I'm looking at it...

Reply:
@ihostage, I did some extra local tests and I believe we are fine. The real motivation for this PR is to remove the directly dependency on log4j 1.x that Lagom was adding explicitly. 

In itself, this is not an issue because the brokers impl in question are used in dev-mode. 

The fact is, it's even not clear if log4j 1.x has been impacted, as mentioned [here](https://github.com/apache/logging-log4j2/pull/608#issuecomment-990494126). 
However, it's clear that log4j 1.x won't be updated and there are other vulnerabilities there that won't be fixed. 

I don't even think that Lagom was even using this dependency, because when we define the dependency on the kafka-client, we exclude the `slf4j-log4j12` bridge.
```
val `kafka-client` = libraryDependencies ++= Seq(
    "org.slf4j" % "log4j-over-slf4j" % Versions.Slf4j,
    akkaStreamKafka.exclude("org.slf4j", "slf4j-log4j12"),
```

The kafka-client uses sfl4j and we explicitly removing the bridge to log4j12. 
It's not clear for me why we were explicitly adding log4j 1.2.17 to the client. This was probably needed at some point in the past.

Reply:
@ihostage, I went one step further and even removed the deps. We don't need it at all.

Reply:
I like it 👍 
I love PR's that remove something unused. Code becomes more clear 😄